### DISCUSSION TITLE
Add tools similar to worldedit

### DISCUSSION THREAD

**Discussion by MisledWater79** [Mojang/minecraft-editor#10] [Feedback]
I assume you already plan to do this since we have selection and a brush. But I would like there to be different shapes for the brush and maybe make it so we can make custom brushes. And also the ability to add multiple block types and percentages for them to add randomness.

**Comment by minecraftchloe** [comment]
Yes! We are planning on adding: 
1. Additional brush shapes (eg. sphere, pyramid, cylinder)
2. Custom brush creation - when we release the API update, that'll make it possible for you to make really cool brushes

Where we want to do a bit more scoping + completely agree they would be extremely cool: multiple block types (UI/UX of this is going to be fun!) and percentages for randomness.

**Reply by MisledWater79** [reply]
Alright thanks! I also think painting would be nice, so we could paint certain blocks. Or something like making it replace all blocks but air or but water etc.

**Reply by hmm-monke** [reply]
Built in terrain tools like eroding, lifting, flattening, painting would be nice
